pkg/shell/superuser.jsx

```diff
@@ -303,7 +303,7 @@ const SuperuserDialogs = ({ superuser_proxy, host, create_trigger }) => {
         return;
 
     const show = (superuser_proxy.Current != "root" && superuser_proxy.Current != "init" &&
-                  superuser_proxy.Bridges.length > 0);
+                  (superuser_proxy.Bridges?.length ?? 0) > 0);
```

Comment:
FTR: this isn't strictly necessary, as `undefined > 0 === false`, but these kinds of comparisons make my head hurt, and I'd rather not rely on them.

Reply:
As a graybeard, I would prefer to read this is a `superuser_proxy.Bridges && superuser_proxy.Bridges.length > 0`. All these question marks make my head hurt a bit. :-) But I get used to it I guess. Or can we guard the whole thing with `superuser_proxy.valid && ...`?